### DISCUSSION TITLE
Is there any way to update labels/text of `RadioSet` widget?

### DISCUSSION THREAD

**Discussion by sambitnayak** [Textualize/textual#3764] [General]
A newbie question...

Let's imagine a quiz-type dialog using a `Label`/`Static` for the question and a `RadioSet` for the answer options.
Just like using `Label`/`Static` `update()` to change the question, is there a way to change the labels/text of the `RadioSet`?
Or is it better to remove existing `RadioSet` and create a new one for the new options?

Thanks.

**Comment by davep** [comment]
There isn't currently a method of changing a `RadioButton`'s label so, yes, I would imagine the best approach right now would be to remove the existing one and mount a fresh one.

**Comment by davep** [comment]
Just to follow up: it seemed like a sensible thing to want to do, so we've added the ability to assign to the `label` of a `Checkbox` or a `RadioButton` for the next release: https://github.com/Textualize/textual/pull/3765

So from that release, assuming you give your buttons their own ID, you can query them back and assign the new label:

```python
from textual.app import App, ComposeResult
from textual.widgets import RadioSet, RadioButton

class RadioLabelsApp(App[None]):

    BINDINGS = [("s", "swap")]

    def compose(self) -> ComposeResult:
        yield RadioSet(
            RadioButton("Alpha", id="one"),
            RadioButton("Beta", id="two")
        )

    def action_swap(self) -> None:
        self.query_one("#one", RadioButton).label = "Gamma"
        self.query_one("#two", RadioButton).label = "Delta"

if __name__ == "__main__":
    RadioLabelsApp().run()
```
